Vital check code/Vital.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+namespace VitalAssignment
+{
+    public delegate void VitalAlert(string VitalName, string message);
+    class Vital    
+    {
+        String vitalName;
+        int lowerLimit;
+        int upperLimit;
+        public Vital(String vitalName, int lowerLimit, int upperLimit)
+        {
+            this.vitalName = vitalName;
+            this.lowerLimit = lowerLimit;
+            this.upperLimit = upperLimit;
+        }
+
+        public bool isVitalOk(int vitalValue, VitalAlert alertType)
+        {
+           
+            if (vitalValue < lowerLimit)
+            {
+                VitalAlert alert;
+                alert = alertType;
+                string message = "is LOW from the required range";
+                alert.Invoke(vitalName, message);
```

Comment:
Can also directly call Invoke on the alertType?

Reply:
I have called Invoke directly on alertType. I did not know it was possible. Thank you.